CMakeLists.txt

```diff
@@ -40,7 +40,6 @@ if (JSON_LIB)
 else (JSON_LIB)
 	message (WARNING "JSON library *NOT* found. JSON support *disabled*!")
 endif (JSON_LIB)
-target_compile_options(frugen PUBLIC -m32)
```

Comment:
I believe this change does not belong to this commit. The line may not be the best one, but removing it just switches the project to host default architecture, which is not what I want. Adding a proper way to choose the target architecture should go in a separate commit, not along with fixing C-compiler warnings.